subprojects/parseq-restli-client/src/main/java/com/linkedin/restli/client/GetRequestGroup.java

```diff
@@ -434,6 +445,11 @@ public boolean equals(Object obj) {
         return false;
     } else if (!_headers.equals(other._headers))
       return false;
+    if (_cookies == null) {
```

Comment:
You can use Objects.equals here (https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#equals-java.lang.Object-java.lang.Object-)

Reply:
Good call. Refactored